tokio-net/src/uds/datagram.rs

```diff
@@ -75,6 +75,11 @@ impl UnixDatagram {
         poll_fn(|cx| self.poll_send_priv(cx, buf)).await
     }
 
+    /// Try to send a datagram from the socket without blocking.
+    pub fn try_send(&mut self, buf: &[u8]) -> io::Result<usize> {
+        self.io.get_mut().send(buf)
```

Comment:
```suggestion
        self.io.get_ref().send(buf)
```

Should be okay here!